pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -835,6 +835,13 @@ func (m *kubeGenericRuntimeManager) GetPodStatus(uid kubetypes.UID, name, namesp
 	})
 	glog.V(4).Infof("getSandboxIDByPodUID got sandbox IDs %q for pod %q", podSandboxIDs, podFullName)
 
+	// Get statuses of all containers visible in the pod.
```

Comment:
This is to trigger 'ContainerStatus' call first, which in dockershim saves podIP to cache, Then later in call to 'PodSandboxStatus' it was able to retrieve that saved podIP.
https://github.com/kubernetes/kubernetes/pull/46744/files#diff-c7dd39479fd733354254e70845075db5R382
https://github.com/kubernetes/kubernetes/pull/46744/files#diff-8cfbccadb55fb07d421430cd1779e7a2R386

This is a workaround for the case where sandbox is not in use, while podIP comes from container.